app/subscriber/src/features/login/UnauthenticatedHome.tsx

```diff
@@ -58,7 +58,7 @@ export const UnauthenticatedHome: React.FC<IUnauthenticatedHomeProps> = (props)
   React.useEffect(() => {
     const mobileMediaQuery = window.matchMedia('(max-width: 767px)'); // Adjust the breakpoint as needed
 
-    const handleMobileChange = (event: any) => {
+    const handleMobileChange = (event: MediaQueryListEvent) => {
```

Comment:
Thankyou

Reply:
Np! Update the PR with a few more things now